src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,30 +63,25 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    // Schedule `ExampleCommand` when `exampleCondition` changes to `true`
-    //configure when have commands
-    // new JoystickButton(_operator, JoystickConstants.BUMPER_RIGHT)
-    //   .onTrue(new InstantCommand(() -> _bootyIntake.setState(BootyState.CubeIntake)))
-    //   .onFalse(new InstantCommand(() -> _bootyIntake.setState(BootyState.CubeHold))); 
-    // new JoystickButton(_operator, JoystickConstants.BUMPER_LEFT)
-    //   .onTrue(new InstantCommand(() -> _bootyIntake.setState(BootyState.ConeIntake)))
-    //   .onFalse(new InstantCommand(() -> _bootyIntake.setState(BootyState.ConeHold)));
+    _operator
+      .leftBumper()
+      .whileTrue(new RunCommand(_leftClaw::ClawUp, _leftClaw));
+      //.whileTrue(Commands.runOnce(() -> _leftClaw.ClawUp(), _leftClaw));
     
-    new JoystickButton(_operator, JoystickConstants.BUMPER_RIGHT).whileTrue(new RunCommand(_rightClaw::rightClawUp, _rightClaw));
-    new JoystickButton(_operator, JoystickConstants.RIGHT_TRIGGER).whileTrue(new RunCommand(_rightClaw::rightClawDown,_rightClaw));
-    new JoystickButton(_operator, JoystickConstants.BUMPER_LEFT).whileTrue(new RunCommand(_leftClaw::leftClawUp, _leftClaw));
-    new JoystickButton(_operator, JoystickConstants.LEFT_TRIGGER).whileTrue(new RunCommand(_leftClaw::leftClawDown, _leftClaw));
-      //new JoystickButton(_operator, JoystickConstants.Y).whileTrue(new RunCommand(_fourBarArms::armOut, _fourBarArms));
-    // new JoystickButton(_operator, JoystickConstants.A).whileTrue(new RunCommand(_fourBarArms::armIn, _fourBarArms));
-    // new JoystickButton(_operator, JoystickConstants.X).whileTrue(new RunCommand(_intakePivot::pivotUp, _intakePivot));
-    // new JoystickButton(_operator, JoystickConstants.B).whileTrue(new RunCommand(_intakePivot::pivotDown, _intakePivot));
+    _operator
+      .rightBumper()
+      .whileTrue(new RunCommand(_rightClaw::ClawUp, _rightClaw));
+      //.whileTrue(Commands.runOnce(() -> _rightClaw.ClawUp(), _rightClaw));
     
-    // new Trigger(m_dualClaw::exampleCondition)
-    //     .onTrue(new LeftClaw(m_dualClaw));
-
-    // // Schedule `exampleMethodCommand` when the Xbox controller's B button is pressed,
-    // // cancelling on release.
-    // m_driverController.b().whileTrue(m_dualClaw.exampleMethodCommand());
+    _operator
+      .leftTrigger()
```

Comment:
since we're just using the default 0.5 threshold for this function, should we get rid of `leftTriggerThreshold = 0.4;`?